### PR TITLE
Patch 13

### PR DESCRIPTION
**Pull request by KVDD**
Please prefix your pull request with one of the following: **[FEATURE]** **[FIX]** **[IMPROVEMENT]**.

**In raising this pull request, I confirm the following (please check boxes):**

- [ ] I have read and understood the [contributors guide](https://github.com/CCExtractor/ccextractor/blob/master/.github/CONTRIBUTING.md).
- [ ] I have checked that another pull request for this purpose does not exist.
- [ ] I have considered, and confirmed that this submission will be valuable to others.
- [ ] I accept that this submission may not be used, and the pull request closed at the will of the maintainer.
- [ ] I give this submission freely, and claim no ownership to its content.

**My familiarity with the project is as follows (check one):**

- [ ] I have never used CCExtractor.
- [ ] I have used CCExtractor just a couple of times.
- [ ] I absolutely love CCExtractor, but have not contributed previously.
- [ ] I am an active contributor to CCExtractor.

---

{pull request content here}

### CODE REVIEW

File: Dictionary/dict_DesignatedSurvivor.txt [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+ Aaron Shore 
+ Emily Rhodes 
+ Mike Ritter 
+ John Forstell 
+ Jason Atwood 
+ Leo Kirkman
+ Chuck Russink 
+ Kimble Hookstraten 
+ Penny Kirkman 
+Lyor Boone
```
Comment (saurabhshri) [inline Dictionary/dict_DesignatedSurvivor.txt]: All these whitespaces before the words are not desirable.

File: dict_DesignatedSurvivor [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+ Majid Nassar 
+ Eva Booker 
+Charlie Langdon 
+ Lisa Jordan 
+ Nadia Espinosa 
+ Tiffany Gimble 
+ Dax Minter 
+ Admiral Chernow 
+ Peyton Lane 
+ Detective Blakey
```
Comment (saurabhshri) [inline dict_DesignatedSurvivor]: Please sort the words alphabetically. It's a dictionary file after all and the algorithm relies on it. Applicable on all files you added.

File: dict_DesignatedSurvivor [hunk 3]
```diff
@@ -0,0 +1,40 @@
+Tom Kirkman
```
Comment (saurabhshri) [inline dict_DesignatedSurvivor]: Please put it in correct directory as others.

**Comment by saurabhshri** [comment]
Also, please edit the PR title to reflect the proposed change and tick all the required checkboxes in PR description. Without it - the PR can not be merged.

**Comment by canihavesomecoffee** [comment]
Closing this. Please re-open on https://github.com/CCExtractor/show_specific_dictionaries, and make sure you follow *ALL* requirements (those include the remarks from Saurabh above).

**Pull request closed without merging**